package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "homepage": "https://MediaComem.github.io/bytesim-webapp",
   "dependencies": {
+    "@chakra-ui/icons": "^2.0.11",
```

Comment:
could you ask yoann for the icon you used from chakra-ui to provide you svgs? We try to use our own icon for consistency and to avoir adding an optional package

Reply:
ok I asked Yoann

Reply:
nice, could you check the conflicts and fix them?